Makefile

```diff
@@ -0,0 +1,61 @@
+IMAGE_NAME = canopy
+DOCKERFILE_DIR = .
+COMMON_BUILD_ARGS = --progress plain
+EXTRA_BUILD_ARGS =
+IMAGE_VERSION = $(shell poetry version -s)
+PORT = 8000
+ENV_FILE = .env
+
+.PHONY: lint static test test-unit test-system test-e2e docker-build docker-build-dev docker-run docker-run-dev help
+
+lint:
+	poetry run flake8 .
+
+static:
+	poetry run mypy src
+
+test:
+	poetry run pytest -n 8 --dist loadscope
```

Comment:
The number of workers should probably be a param (with default)

Reply:
Done